docker.local/config/0chain.yaml

```diff
@@ -48,32 +49,33 @@ server_chain:
     max_block_size: 10
     max_block_cost: 10000 #equal to 100ms
     max_byte_size: 1638400
+    generators: 2
```

Comment:
What does the `generators` do, we are getting generators number base on the `generators_percent` and if it is less than the `min_generators`, then we will use the `min_generators`.

Reply:
the same